### PR TITLE
dibs: I will implement exercise scrabble-score

### PR DESCRIPTION
**Pull request by zgavin1**
- [x] Include test suite from ES6 track

- [x] Write example that passes tests

### CODE REVIEW

**Comment by masters3d** [comment]
Nice! you might have to change the test suit so that it works with Jest. Jasmine and Jest are a little different.

**Comment by zgavin1** [comment]
Hmm - the failure is on the `is-finite` module import.

Also, when I run the full `make` command, the grade-school assignment exercise example fails its tests. Just FYI.

**Comment by masters3d** [comment]
I restarted the build. We may have to upgrade TS.

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (masters3d) [inline yarn.lock]: We don't need this file in the root. Otherwise, looks good to me.

Comment (zgavin1) [inline yarn.lock]: right - accidentally ran yarn install in the directory in my frantic attempt to fix the build.

**Comment by zgavin1** [comment]
All set. Thanks for the assist on the build restart @masters3d

**Pull request merged**